source/server/configuration_impl.cc

```diff
@@ -193,24 +193,10 @@ WatchdogImpl::WatchdogImpl(const envoy::config::bootstrap::v3::Watchdog& watchdo
 }
 
 InitialImpl::InitialImpl(const envoy::config::bootstrap::v3::Bootstrap& bootstrap,
-                         const Options& options, Instance& server)
+                         const Options& options, Instance&)
```

Comment:
Why did you decide to keep `Instance` parameter around instead of removing it?

Reply:
maybe almost `InitialImpl` test use it, ok, i remove it